test/repr/main.ml

```diff
@@ -7,6 +7,8 @@ module Alcotest = struct
     let equal = T.(unstage (equal typ)) in
     let pp = T.pp_dump typ in
     check ?pos (testable pp equal) msg a b
+
+  let string = testable Fmt.Dump.string String.equal
```

Comment:
just curious, why override this? it does not seem to make a difference in the tests.

Reply:
Alcotest uses `Fmt.string` (which is `"%s"` internally), whereas `Fmt.Dump.string` is `"%S"`. The latter adds proper escapes for weird bit sequences, which ended up being useful here because Repr can add length headers that don't render correctly in the test output otherwise.

(I think Alcotest should probably default to `Fmt.Dump.string` instead.)